src/worker.cpp

```diff
@@ -1393,4 +1393,99 @@ void Worker::performBackupAndRestore(types::VPDMapVariant& io_srcVpdMap)
         */
     }
 }
+
+void Worker::deleteFruVpd(const std::string& i_dbusObjPath)
+{
+    try
+    {
+        if (m_parsedJson.empty())
+        {
+            throw std::runtime_error(
+                "System config JSON is empty. Cannot perform deletion for: " +
+                i_dbusObjPath);
+        }
+
+        const std::string& l_fruPath =
```

Comment:
Where are you using this variable in this flow? If not why do you even need this.

Reply:
It is not being used. Removed